### PR TITLE
Add integration test for MultiUserChat

### PR DESCRIPTION
**Pull request by adiaholic**
SMACK-888 has a mention of unreachable code and is now solved under commit 0f7b7df1f0cd6a3d2f75a983143d02fda8990111. This integration test tests those class entities which were to be set by previously stated unreachable
code.

### CODE REVIEW

File: smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatIntegrationTest.java [hunk 1]
```diff
@@ -120,4 +123,88 @@ public void processMessage(Message message) {
...
+
+        EntityBareJid mucAddress = JidCreate.entityBareFrom(Localpart.from("smack-inttest-join-leave-" + randomString),
+                                                            mucService.getDomain());
+
+        MultiUserChat muc = mucManagerOne.getMultiUserChat(mucAddress);
+        muc.join(Resourcepart.from("nick-one"));
+
+        final SimpleResultSyncPoint mucDestroyed = new SimpleResultSyncPoint();
+
+        UserStatusListener userStatusListener = new UserStatusListener() {
```
Comment (Flowdalic) [inline smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatIntegrationTest.java:138]: If you instantiate a `DefaultUserStatusListener` then you could omit all those empty methods.

Comment (adiaholic) [reply smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatIntegrationTest.java]: Replaced `UserStatusListener` with `DefaultUserStatusListener`.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 4313b46aead627a805ca60437c03c1a677210feb
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/igniterealtime/Smack/wiki/Guidelines-for-Smack-Developers-and-Contributors

---
This message was auto-generated by https://gitcop.com

**Comment by Flowdalic** [comment]
Thanks, applied as https://github.com/igniterealtime/Smack/commit/1d49de6b60052f641505c20e1c72855ccb727516

**Pull request closed without merging**